scss/_base_typography-definitions.scss

```diff
@@ -218,17 +218,20 @@
     }
   }
 
+  %muted-text {
+    color: $colors--light-theme--text-muted;
+  }
+
   %table-header-label {
     @extend %x-small-text;
 
```

Comment:
Did you mean to remove color value for this placeholder?

Reply:
not sure what you mean?

Reply:
Previously table headers were muted and this removed that and makes them color: #111;

Reply:
ah yes, just noticed this myself, I've updated it.